custom_components/variable/__init__.py

```
@@ -63,14 +63,16 @@ def async_remove_helper_devices(
         helper_config_entry_id: str,
         source_device_id: str | None,
         remove_all_devices: bool = False,
+        keep_device_ids: Collection[str] = (),
```

Ruff (0.16.1)

[warning] Unused function argument: `keep_device_ids`

(ARG001)